.env-template

```diff
@@ -26,6 +26,11 @@ JWT_SECRET=
 JWT_REFRESH_SECRET=
 JWT_TOKEN_EXPIRY=
 
+## Community Engagement - How actions are weighted to determine engagement in a community
+COMMUNITY_ENGAGEMENT_WEIGHT_CREATE_COMMUNITY=
+COMMUNITY_ENGAGEMENT_WEIGHT_CREATE_POST=
+COMMUNITY_ENGAGEMENT_WEIGHT_CREATE_COMMENT=
```

Comment:
Love the weighting of community engagement. Just curious, why is it an environment variable?

Reply:
So that it can be configured by the client for the business needs.